testem.js

```diff
@@ -10,17 +10,13 @@ module.exports = {
   ],
   browser_args: {
     Chrome: {
-      mode: 'ci',
-      args: [
```

Comment:
The `mode, args` approach is officially deprecated by `testem`. 

https://github.com/testem/testem/blob/master/docs/browser_args.md

Also, we had two `Chrome` keys in the object, which doesn't really work and likely meant the second one was being used in all cases.